### ISSUE TITLE
Can't use auto with agda 2.6.0

### ISSUE THREAD

**Issue by Zinfour** [banacorn/agda-mode#91]
I just tried to replace my agda installation with the master branch and now when I use ```C-c C-a``` I get
```
agda2-parse-error cannot read: IOTCM "/path/to/file.agda" NonInteractive Direct ( Cmd_auto 0 (intervalsToRange (Just (mkAbsolute "/path/to/file.agda")) [Interval (Pn () 7 1 7) (Pn () 10 1 10)]) "" )

**Comment by banacorn** [comment]
Thanks, we are working on a patch! https://github.com/banacorn/agda-mode/pull/88

**Comment by banacorn** [comment]
Should be fixed on `v0.8.10` now
